doc/user_guide.md

```diff
@@ -0,0 +1,324 @@
+
+The microsoft documentation on deploying azure functions is way more exhaustive so if this doesn't cover your needs you might find a better option there.
+
+Let's get started:
+
+### Zip Deployment using the Azure Portal and Azure CLI:
+
+Setting everything up in the azure portal:
+
+Navigate to the azure portal and log in if you haven't done so yet: [Microsoft Azure](https://portal.azure.com/#home)
+
+On the dashboard, click on 'Create a resource'.
+
+![](./img/2021-08-04-15-02-42-image.png)
+
+Search for 'function', then pick 'Function App'.
+
+![](./img/2021-08-04-15-04-40-image.png)
+
+Click 'Create'.
+
+![](./img/2021-08-04-15-05-43-image.png)
+
+Now we need to do some setup:
+
+![](./img/2021-08-04-15-06-27-image.png)
+
+Create a new resource group (or use an existing one if you want). Take note of the name.
+
+Pick a (unique) function app name.
+
+Pick '.NET' as the runtime stack and pick '3.1' as the version.
+
+Pick the region that fits you the most (this will probably the same region as where you host the Azure Data Factory as well).
+
+![](./img/2021-08-04-15-08-55-image.png)
+
+Next, press 'Hosting'. This will bring you to the next setup screen.
+
+![](./img/2021-08-04-15-11-27-image.png)
+
+Create a new storage account name (recommended) or use an existing one. 
+
+Take note of the name.
+
+As operating system pick 'Windows', ()this is because we use a .dll from the Exasol ADO .NET data provider that's not crossplatform.)
+
+Press 'Review + Create' and then press ''Create'.
+
+You'll get a notification saying 'Deployment is underway'. This might take a minute.
+
+Once the deployment is complete you'll get a notification and you can navigate to the resource. Do so.
+
+![](./img/2021-08-04-15-14-28-image.png)
+
+We've now succesfully created our azure functions app. You can already take a look around. 
+
+As you see there are no functions yet so our next step will be uploading these functions.
+
+For this we'll use the Azure CLI tools.
+
+#### Install the Azure CLI (get it here):
+
+[Install the Azure CLI for Windows | Microsoft Docs](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-windows?tabs=azure-cli)
+
+#### Get the zipped release of the Exasol azure helper functions
+
+Get the packaged version of the exasol azure data factory functions here:
+
+Release.zip (TODO: add url)
+
+#### Deploy the zipped functions using the azure CLI
+
+Using the provided .zip in the release you can easily deploy the azure functions.
+
+Download the .zip
+
+Use azure cli:
+
+Login:
+
+Open a command line window ("cmd") or powershell or a terminal of your choice
+
+Type:
+
+az login
+
+Your preferred browser will open and you will be able to login.
+
+Now navigate to the folder where you downloaded the packaged zip file and run the following command:
+
+az webapp deployment source config-zip -g $resourceGroupName -n $functionName --src $sourceZipPath
```

Comment:
I think this would look better with a code block. The same suggestion for all other commands

Reply:
done